to_do/project_todo/todo_command_line_pr/alt_main_command_line.py

```diff
@@ -0,0 +1,17 @@
+import sys
+from todolist import ToDoList
+
+#Question: what if user enters multiple todos on command line? How to capture these? Need to specifiy a var for each todo entered on command line?
```

Comment:
@paulghaddad Pls note this question in this version of sys.arg: what if user enters multiple todos on command line?